BackofficeBundle/Manager/RedirectionManager.php

```diff
@@ -68,4 +74,21 @@ public function createRedirection($pattern, $nodeId, $language)
             }
         }
     }
+
+    /**
+     * @param string $nodeId
+     * @param int    $version
+     * @param string $language
+     */
+    public function deleteRedirection($nodeId, $language)
+    {
+        $redirections = $this->redirectionRepository->findByNode($nodeId, $language);
+        if (count($redirections) > 0) {
```

Comment:
the count here is useless, if the `array` returned by the repository is empty, there will be no error